### DISCUSSION TITLE
How to update state of editor without Re-render

### DISCUSSION THREAD

**Discussion by levi-katarok** [steven-tey/novel#282] [Q&A]
Hi All, 

I've poured over all the discussions and code I could here but haven't been able to figured out/find out how to just update any or all of the current content in the Editor React component without forcing a re-render. I just want to stream in text and it's stumped me! 

Any help is greatly appreciated. For reference this is how I'm handling it now

` setEditorContent(prevContent => prevContent + chunksBuffer);`

but every chunk forces a clunky re-render

**Comment by mhzrerfani** [comment]
https://tiptap.dev/docs/editor/api/commands/content/set-content
